processes/ripple_billing.js

```diff
@@ -19,9 +19,90 @@
 module.exports = function(codius) {
 
   if (codius.features.isEnabled('RIPPLE_BILLING')) {
-    if (!codius.config.get('RIPPLE_ADDRESS')) {
+    codius.logger.info("Ripple Billing Feature Enabled")
+
+    const RippleAccountMonitor = require('ripple-account-monitor')
+    const http                 = require('superagent')
+    const Promise              = require('bluebird')
+    const RIPPLE_REST_URL      = 'https://api.ripple.com/'
+    const ADDRESS              = codius.config.get('RIPPLE_ADDRESS')
+    const billing              = new codius.BillingService()
+    const CPU_PER_DROP         = 0.0075
+    const CPU_PER_BITCOIN      = 100000000
```

Comment:
Should this use `compute_units_per_bitcoin` from config.js? 
If so, also add `compute_units_per_drop` to the config.